matt.diff

```diff
@@ -0,0 +1,48 @@
+diff --git a/lightning/src/ln/channel.rs b/lightning/src/ln/channel.rs
```

Comment:
This proabbly doesn't need to be in git :)

Reply:
OOP haha

Reply:
git status is a very handy command, i should try it sometime